src/main/java/fi/otavanopisto/kuntaapi/server/integrations/ptv/updaters/PtvServiceChannelEntityUpdater.java

```diff
@@ -404,4 +422,15 @@ private void indexServiceLocationChannel(Long orderIndex, ServiceLocationService
     }
   }
 
+  private void updateParentPageIds(List<PageId> parentPageIds) {
+    for (PageId parentPageId : parentPageIds) {
+      Identifier parentPageIdentifier = identifierController.findIdentifierById(parentPageId);
+      if (ManagementConsts.IDENTIFIER_NAME.equals(parentPageIdentifier.getSource())) {
+        // TODO: REFACTOR THIS AWAY FROM THE PTV INTEGRATION PLUGIN
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)